### PR TITLE
#164295757 fix navigation display error

### PR DESCRIPTION
**Pull request by kwameasiago**
#### What does this PR do?
Fix the navigation bar to display the appropriate links

#### Description of Task to be completed?
Add a logout feature
Fix navigation bar to show the correct links

#### How should this be manually tested?
- Clone the repo locally.
- Checkout to the `bg-navigation-bar-fix-164295757`.
- Run `npm install` to install all dependencies.
- Navigate to the home page view the navbar
#### What are the relevant pivotal tracker stories?
[#164295757](https://www.pivotaltracker.com/story/show/164295757)

### CODE REVIEW

**Comment by andrewhingah** [comment]
Hi @SelaDanti 
When I manually test this, the token is still available in the local storage after sign out. Kindly fix the problem.

**Review by andrewhingah — COMMENTED** [review]
Hi @SelaDanti
When I manually test this, the token is still available in the local storage after sign out. Kindly fix the problem.

File: src/components/NavBar/index.jsx [hunk 1]
```diff
@@ -1,69 +1,80 @@
...
-      loggedIn: false
-    };
-  }
-
-  toggleNavBar = () => {
-    this.setState({
-      isOpen: !this.state.isOpen
-    });
+export class NavBar extends Component {
+  clearToken = (dispatch) => {
```
Comment (cosmas28) [inline src/components/NavBar/index.jsx]: Kindly move this to the `mapDispatchToProps`. Also clear localStorage in the logout action.

Comment (kwameasiago) [reply src/components/NavBar/index.jsx]: ok

**Review by cosmas28 — APPROVED** [review]
LGTM

**Pull request merged**